### PR TITLE
Changed erb to slim

### PR DESCRIPTION
**Pull request by alebianwolox**
## Summery
Changed html.erb files format to slim format
## Test Plan
Go to every page of the app
## Trello Card
https://trello.com/c/6kkAAmyQ

### CODE REVIEW

File: app/views/articles/_form.slim [hunk 1]
```diff
@@ -0,0 +1,20 @@
+= form_for @article do |f|
+  - if @article.errors.any?
+    #error_explanation
+      h2
+        = pluralize(@article.errors.count, "error")
+        | prohibited
+        | this article from being saved:
+      ul
+        - @article.errors.full_messages.each do |msg|
+          li= msg
```
Comment (JesusHerrera) [inline app/views/articles/_form.slim]: Place text and tags in different lines

File: app/views/articles/_form.slim [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  p
+    = f.label :title
+    br/
```
Comment (JesusHerrera) [inline app/views/articles/_form.slim]: Don't use ```br/````to create space between elements. Use CSS to create gaps when required.

File: app/views/articles/_form.slim [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  - if @article.errors.any?
+    #error_explanation
+      h2
+        = pluralize(@article.errors.count, "error")
+        | prohibited
+        | this article from being saved:
+      ul
+        - @article.errors.full_messages.each do |msg|
+          li= msg
+  p
```
Comment (JesusHerrera) [inline app/views/articles/_form.slim]: This is not a paragraph, so ```p``` is not required

File: app/views/articles/_form.slim [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    = f.label :title
+    br/
+    = f.text_field :title
+  p
```
Comment (JesusHerrera) [inline app/views/articles/_form.slim]: This is not a paragraph, so ```p``` is not required

File: app/views/articles/_form.slim [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    = f.label :text
+    br/
```
Comment (JesusHerrera) [inline app/views/articles/_form.slim]: Don't use ```br/``` to create space between elements. Use CSS to create gaps when required.

File: app/views/articles/_form.slim [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    = f.text_area :text
+  p
```
Comment (JesusHerrera) [inline app/views/articles/_form.slim]: This is not a paragraph, so ```p``` is not required

File: app/views/articles/edit.slim [hunk 7]
```diff
@@ -0,0 +1,5 @@
+h1 Edit article
+
+= render 'form'
+ 
+= link_to 'Back', articles_path
```
Comment (JesusHerrera) [inline app/views/articles/edit.slim]: Git requires an empty line at the end of files

File: app/views/articles/index.slim [hunk 8]
```diff
@@ -0,0 +1,21 @@
+h1 Listing articles
+= link_to 'New article', new_article_path
+table
```
Comment (JesusHerrera) [inline app/views/articles/index.slim]: When using a ```table``` use ```thead``` and ```tbody```

File: app/views/articles/index.slim [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+      td
+        = article.title
+      td
+        = article.text
+      td
+        = link_to 'Show', article_path(article)
+      td
+        = link_to 'Edit', edit_article_path(article)
+      td
+        = link_to('Delete', article_path(article), method: :delete, data: { confirm: 'Are you sure?' })
```
Comment (JesusHerrera) [inline app/views/articles/index.slim]: Git requires an empty line at the end of files

File: app/views/articles/show.slim [hunk 10]
```diff
@@ -0,0 +1,15 @@
+p
```
Comment (JesusHerrera) [inline app/views/articles/show.slim]: This is not a paragraph, so ```p``` is not required

File: app/views/articles/show.slim [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+  strong Title:
```
Comment (JesusHerrera) [inline app/views/articles/show.slim]: Only use ```strong``` when highlighting a text within a ```p```

File: app/views/articles/show.slim [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+p
+  strong Text:
+  = @article.text
+h2 Comments:
+= render @article.comments
+h2 Add a comment:
+= render 'comments/form'
+= link_to 'Edit', edit_article_path(@article)
+| \
+td= link_to('Delete article', article_path(@article), method: :delete, data: { confirm: 'Are you sure?' } )
```
Comment (JesusHerrera) [inline app/views/articles/show.slim]: ```td``` is table data, a cell inside a table.

File: app/views/comments/_comment.slim [hunk 13]
```diff
@@ -0,0 +1,11 @@
+p
```
Comment (JesusHerrera) [inline app/views/comments/_comment.slim]: Fix identation issues, removed unnecessary tags and delete empty lines

File: app/views/comments/_form.slim [hunk 14]
```diff
@@ -0,0 +1,11 @@
+= form_for([@article, @article.comments.build]) do |f|
```
Comment (JesusHerrera) [inline app/views/comments/_form.slim]: Fix identation issues, removed unnecessary tags and delete empty lines

File: app/views/layouts/application.slim [hunk 15]
```diff
@@ -0,0 +1,10 @@
+doctype html
```
Comment (JesusHerrera) [inline app/views/layouts/application.slim]: Fix identation issues

**Comment by alebianwolox** [comment]
@JesusHerrera Fixed errors

**Comment by JesusHerrera** [comment]
:+1:

**Pull request merged**